### PR TITLE
Cogboard authentication

### PR DESCRIPTION
**Pull request by szymon-owczarzak**

### CODE REVIEW

File: cogboard-webapp/src/components/UserLogin.js [hunk 1]
```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+
+import { TextField, Button, IconButton, Typography } from '@material-ui/core';
+import { AccountCircle, PowerSettingsNew } from '@material-ui/icons';
+import { useDialogToggle, useFormData } from '../hooks';
```
Comment (goeson00) [inline cogboard-webapp/src/components/UserLogin.js]: Move thunks and hooks imports above the component imports. Generally we have 3 types of imports in the following order separated by line breaks: libs, own helpers, hooks etc. and components (3rd party components and own).

File: cogboard-webapp/src/components/UserLogin.js [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+import { StyledFieldset } from './styled';
+import { login, logout } from '../actions/thunks';
+import AppDialog from './AppDialog';
+
+const UserLogin = () => {
+
```
Comment (goeson00) [inline cogboard-webapp/src/components/UserLogin.js]: Remove empty line.

File: cogboard-webapp/src/components/UserLogin.js [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+  const dispatch = useDispatch();
+  const {values, handleChange} = useFormData({});
+  const [dialogOpened, openDialog, handleDialogClose] = useDialogToggle();
+  const errorMsg = useSelector(({app}) => app.loginErrorMessage);
+  const jwToken = useSelector(({app}) => app.jwToken);
+
+  const submitCredentials = (credentials) => () => {
```
Comment (goeson00) [inline cogboard-webapp/src/components/UserLogin.js]: I suggest handleLoginButtonClick name for click handler.

File: cogboard-webapp/src/reducers/app/isAdmin.js [hunk 4]
```diff
@@ -0,0 +1,7 @@
+import { LOGIN_SUCCESS } from '../../actions/types';
+
+const isAdmin = (state = false, { type, payload }) => {
+  return type === LOGIN_SUCCESS ? payload !== '' : state;
```
Comment (goeson00) [inline cogboard-webapp/src/reducers/app/isAdmin.js]: Change payload !== '' to true;

Comment (szymon-owczarzak) [reply cogboard-webapp/src/reducers/app/isAdmin.js]: it does not work this way

**Pull request merged**